src/map.js

```diff
@@ -106,11 +106,32 @@ class Map {
 	isExplored( x, y ) {
 		return this._tiles[x][y].getIsExplored();
 	}
+
+	getStartingCoordinates(x, y) {
+		let startingX = x - Math.round(Lootr.getGameDisplayWidth()/2) + 1;
+		const maxStartingX = this.getWidth() - Lootr.getGameDisplayWidth();
+		if (startingX <= 0) {
```

Comment:
Since there seems to be no `else` as an option here, a ternary might work great.

```
startingX = (startingX <= 0) ? 0 : maxStartingX;